### PR TITLE
Tickets/dm34474: Merge Disabled and ConfigApplied tests

### PR DESCRIPTION
**Pull request by rbovill**
* Merged the Disabled and ConfigApplied tests.
* Improved the 'Verify ConfigurationApplied' and 'Verify ConfigurationsAvailable' keywords in Common_Keywords.resource.
* Added the 'ConfigurationsAvailable Event' tests for the non-config CSCs, now that keyword handles those CSCs.

### CODE REVIEW

**Comment by rbovill** [comment]
This ticket requires more work.

File: Common_Keywords.resource [hunk 1]
```diff
@@ -18,28 +19,59 @@ Verify ConfigurationApplied
     IF    $csc in $NON_CONFIG    
         Should Contain    ${output}    Empty DataFrame
     ELSE
-        Log    ${output}
-        #${line}=    Get Line    ${output}    -1
-        #@{words}=    Split String    ${line}
-        #Log Many    ${words}[2]    ${words}[3]
-        #@{time}=    Split String    ${words}[3]    +
-        #Set Suite Variable    ${watcher_settingsapplied_event_time}    ${time}[0]
+        ${line}=    Get Line    ${output}    -1
```
Comment (mareuter) [inline Common_Keywords.resource:23]: Comment: I think I recall these two lines or something similar showing up in the test reporting output. These two lines are support for the tests, but not something I want to see in the output. Any way to hide this type of thing?

File: Common_Keywords.resource [hunk 2]
```diff
@@ -18,28 +19,59 @@ Verify ConfigurationApplied
...
-    Log Many    Overrides: ${words}[4]    Version: ${words}[5]    URL: ${words}[6]
-    Should Not Be Empty    ${words}[5]
-    Should Not Be Empty    ${words}[6]
+    IF    $csc in $NON_CONFIG
+        Should Contain    ${output}    Empty DataFrame
+    ELSE
+        # Get Overrides value.
+        ${output}=    Get Recent Samples    ${csc}    ${cav_topic}   ${cav_fields}[1]    1    ${csc_index}
+        ${output}=    Convert to String    ${output}
+        ${line}=    Get Line    ${output}    -1
```
Comment (mareuter) [inline Common_Keywords.resource:44]: Comment: Same thing here for all the supporting lines. I don't like the output being polluted by things that I consider support for the tests. They should "just work".

File: Disabled/Verify_ComCam_Disabled.robot [hunk 3]
```diff
@@ -2,21 +2,40 @@
 Library    QueryEfd
 Library    Collections
 Library    String
+Library    DateTime
+Resource    ../Common_Keywords.resource
 Resource    ../Global_Vars.resource
 Resource    ../CSC_Lists.resource
-Force Tags    disabled
+Force Tags    timing
```
Comment (mareuter) [inline Disabled/Verify_ComCam_Disabled.robot:9]: I think this tag should be `comcam`

File: Disabled/Verify_ObsSys1_Disabled.robot [hunk 4]
```diff
@@ -6,34 +6,36 @@ Library    DateTime
...
 ${time_window}    10
 
 *** Test Cases ***
 Verify ScriptQueue:1 Disabled
-    [Tags]    obsys1
+    [Tags]    disabled
     Verify Summary State    ${STATES}[disabled]    ScriptQueue:1
 
-Verify ScriptQueue:2 Disabled
+Verify ScriptQueue:1 ConfigurationApplied Event
```
Comment (mareuter) [inline Disabled/Verify_ObsSys1_Disabled.robot:19]: Check all the tags in this file.

**Review by mareuter — APPROVED** [review]
Few issues found. I have comments about some of the supporting lines I would like you to consider, but not necessarily now, unless it is easy to deal with. Otherwise, looks fine.

File: CSC_Lists.resource [hunk 5]
```diff
@@ -10,5 +10,5 @@
 @{MTCS}            MTMount    MTPtg    MTDome    MTDomeTrajectory    MTAOS    MTHexapod:1    MTHexapod:2    MTRotator    MTM1M3    MTM2
 @{ObsSys1}         ScriptQueue:1    ScriptQueue:2    Watcher
 @{ObsSys2}         Scheduler:1    Scheduler:2    OCPS:1    OCPS:2
-@{NON_CONFIG}      ATOODS    ATHeaderService    ATMCS    ATPneumatics    ATPtg    CCHeaderService    CCOODS    DSM    LinearStage    MTPtg    ScriptQueue
+@{NON_CONFIG}      ATMCS    ATHeaderService    ATOODS    ATPneumatics    ATPtg    CCHeaderService    CCOODS    DSM    LinearStage    MTPtg    ScriptQueue
```
[the remark this answers is not in the document]
Comment (rbovill) [reply CSC_Lists.resource:13]: Yup.  Fixed.

File: Disabled/Verify_ComCam_Disabled.robot (hunk 3 above)
Comment (rbovill) [reply Disabled/Verify_ComCam_Disabled.robot:9]: Fixed

File: Disabled/Verify_ObsSys1_Disabled.robot (hunk 4 above)
Comment (rbovill) [reply Disabled/Verify_ObsSys1_Disabled.robot:19]: Fixed

File: Common_Keywords.resource (hunk 1 above)
Comment (rbovill) [reply Common_Keywords.resource:23]: I will work on this as part of the report generation ticket.

File: Common_Keywords.resource (hunk 2 above)
Comment (rbovill) [reply Common_Keywords.resource:44]: I will work on this as part of the report generation ticket.

**Pull request merged**